tests/kernel/common/src/clock.c

```diff
@@ -35,7 +35,7 @@ K_TIMER_DEFINE(ktimer, duration_expire, stop_expire);
 static ZTEST_BMEM struct timer_data tdata;
 
 #define DURATION 100
-#define LESS_DURATION 80
+#define LESS_DURATION 70
```

Comment:
This looks fine.  On (non-icount/realtime) emulation platforms, a host scheduling quantum is on the order of 10ms.  So it used to be "2 host ticks", and now it's three.  That wasn't a lot of wiggle room on a loaded CI host, and all the test is doing is making sure that the timer hasn't fired yet anyway.